installer/tue-install-impl.bash

```diff
@@ -378,67 +383,70 @@ function tue-install-add-text
     tue-install-debug "tue-install-add-text $*"
 
     local source_file=$1
+    # shellcheck disable=SC2088
     if [[ "$source_file" == "/"* ]] || [[ "$source_file" == "~/"* ]]
     then
         tue-install-error "tue-install-add-text: Only relative source files to the target directory are allowed"
     else
         source_file="$TUE_INSTALL_CURRENT_TARGET_DIR/$source_file"
     fi
     local target_file=$2
+    # shellcheck disable=SC2088
     if [[ "$target_file" != "/"* ]] && [[ "$source_file" != "~/"* ]]
     then
         tue-install-error "tue-install-add-text: target file needs to be absolute or relative to the home directory"
     fi
 
     local root_required=true
-    if namei -l $target_file | grep -q $(whoami)
+    if namei -l "$target_file" | grep -q "$(whoami)"
     then
         tue-install-debug "tue-install-add-text: NO root required"
         root_required=false
     else
         tue-install-debug "tue-install-add-text: root required"
     fi
 
-    if [ ! -f $source_file ]
+    if [ ! -f "$source_file" ]
     then
         tue-install-error "tue-install-add-text: No such source file: $source_file"
     fi
 
-    if [ ! -f $target_file ]
+    if [ ! -f "$target_file" ]
     then
         tue-install-error "tue-install-add-text: No such target file: $target_file"
     fi
 
-    local begin_tag=$(head -n 1 $source_file)
-    local end_tag=$(awk '/./{line=$0} END{print line}' $source_file)
-    local text=$(sed -e :a -e '/^\n*$/{$d;N;};/\n&/ba' $source_file)
+    local begin_tag end_tag text
```

Comment:
Better to have 3 local declarations for improved readability

Reply:
too much lines